aspnetcore/includes/advancedRP.md

```diff
@@ -1 +1 @@
-For a more advanced introduction aimed at experienced developers, see [Introduction to Razor Pages](xref:razor-pages/index).
+For a more advanced introduction aimed at experienced developers or those familar with controllers and views, see [Introduction to Razor Pages](xref:razor-pages/index).
```

Comment:
Typo familar --> familiar

You could consider omitting "experienced" since that's implied by "familiar with ...":

```suggestion
For a more advanced introduction aimed at developers who are familiar with controllers and views, see [Introduction to Razor Pages](xref:razor-pages/index).
```